test/test.js

```diff
@@ -8,6 +8,18 @@ var Runner = require('../lib/runner');
 var fork = require('../lib/fork');
 var ava = require('../lib/test');
 
+// global Promise for zen-observable
+if (!global.Promise) {
+	Object.defineProperty(global, 'Promise', {
+		value: Promise,
+		configurable: true,
+		enumerable: false,
+		writable: true
+	});
+}
+
+var Observable = require('zen-observable');
+
```

Comment:
I'm unsure of what the goal is here for "Supporting Observables". What do you need them to do? Do you need to support subscription to them? Do you need to support returning them?

If it's the former, you should be able to test for `obj[Symbol.observable]` and call it (like a function) to get back an `observable` object with a `subscribe()` function on it that follows the `zen-observable` spec. For that, there's really no need to import a library.

On the other hand, if you want to emit them, you'll want to use the zen-observable polyfill or the RxJS 5 implementation as they follow the spec. And you'll need to do something like you're doing here.